### DISCUSSION TITLE
Restriction on window width reduction.

### DISCUSSION THREAD

**Discussion by minsk21** [sumatrapdfreader/sumatrapdf#3526] [General]
Can't make window narrower

v 3.4.6

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/26014600/0031b7d9-ef78-48a9-8d0c-04117d00f49d)

**Comment by GitHubRulesOK** [comment]
Hmm its been mentioned before, off-hand cant remember what was the trigger /solution perhaps a bad settings file

there is an absolute minimum of 480 mentioned here https://github.com/sumatrapdfreader/sumatrapdf/issues/3031#issuecomment-1423210563 which seems to be a bit larger when scrollbar is active
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/76861ea0-31ef-4d1a-bb4f-9dc8ed283e67)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6671bace-1088-42c3-822f-cde90a947a22)

**Comment by minsk21** [comment]
The problem was in the settings file from the old version. I deleted and created a new one. The restriction has been lifted
